README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/joseph2rs/Website-Volontaria.svg?branch=master)](https://travis-ci.org/joseph2rs/Website-Volontaria)
+[![Build Status](https://travis-ci.org/Volontaria/Website-Volontaria.svg?branch=develop)](https://travis-ci.org/Volontaria/Website-Volontaria)
```

Comment:
@RignonNoel Should the badge show the build status of the master branch?

Reply:
I thought about that and i think it's better to show the develop because master ALWAYS need to be OK, so an image on the readme is not really interresting. I think it's better to show the develop version to help contributor to fix problem.